include/seqan3/search/fm_index/fm_index.hpp

```diff
@@ -223,7 +204,6 @@ class fm_index
     fm_index & operator=(fm_index rhs) //!< When copy/move assigning, also update internal data structures.
     {
         index = std::move(rhs.index);
-        sigma = std::move(rhs.sigma);
```

Comment:
This is just asking a stupid question, but the sigma of a fm-index never changes?

The only reason why we moved / copied it, was because we could not infer it again from the alphabet type once we constructed it?

Reply:
> This is just asking a stupid question, but the sigma of a fm-index never changes?

Nope, fixed by the alphabet.

> 
> The only reason why we moved / copied it, was because we could not infer it again from the alphabet type once we constructed it?

Exactly.